jck/jtrunner/JavaTestRunner.java

```diff
@@ -267,6 +268,16 @@ public static boolean generateJTB() throws Exception {
 			System.out.println("Unable to find additional excludes list file " + jtxFullPath);
 			jtxFullPath = "";
 		}
+		
+		jtxDevFullPath = jckRoot + File.separator + "excludes" + File.separator + jckVersion + "-dev.jtx";
```

Comment:
Can we offer the option to pass in the full path, since in temurin-compliance case we are not able to add files to jckRoot, so we would have to host the exclude file elsewhere.

Reply:
Sure, I will add a new input variable (e.g. `-customJtx`) to JavasTestRunner for passing in full path to an exclude file.

Reply:
thanks Mesbah!

Reply:
Thanks @Mesbah-Alam, I will run a similar test on the temurin-compliance server!